crfs.go

```diff
@@ -966,11 +973,24 @@ func (h *nodeHandle) ReadDirAll(ctx context.Context) (ents []fuse.Dirent, err er
 //
 // See https://godoc.org/bazil.org/fuse/fs#NodeStringLookuper
 func (n *node) Lookup(ctx context.Context, name string) (fspkg.Node, error) {
+    	if c, ok := n.child[name] ; ok {
```

Comment:
This looks like a data race. The node.Lookup can be called from concurrent goroutines and no mutex is protecting this map read/write.

Reply:
Right. I added mutex to protect children maps in e18ee07.

Reply:
That's not enough. This read is also a race.

Reply:
Thank you for your review. I fixed it on 39ae89f.

Reply:
Run gofmt. No space before ;.